test/suites/drift/suite_test.go

```diff
@@ -209,36 +210,13 @@ var _ = Describe("Drift", func() {
 			nodeClaims := env.EventuallyExpectCreatedNodeClaimCount("==", 3)
 			nodes := env.EventuallyExpectCreatedNodeCount("==", 3)
 			env.EventuallyExpectHealthyPodCount(selector, int(numPods))
-			env.Monitor.Reset() // Reset the monitor so that we can expect a single node to be spun up after drift
```

Comment:
Was this monitor reset wrong?

Reply:
You're not doing any checking on the monitor anywhere after this so there's no reason to call the reset from what I could tell.

Reply:
Discussed offline: Confirmed that this monitor reset isn't needed in any of these tests